### PR TITLE
Basic setup done

### PR DESCRIPTION
**Pull request by JyotiBhambhu**
1. Compose setup done for 1 simple screen
2. Architectural layers defined
2. Simple API call implemented
3. Retrofit integrated

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -53,4 +53,7 @@ dependencies {
     implementation "androidx.compose.material:material:$versions.compose_material_version"
     implementation "androidx.compose.ui:ui-tooling-preview:$versions.compose_ui_version"
 
+    implementation 'com.squareup.retrofit2:retrofit:2.9.0'
```
Comment (PrateekDev121) [inline app/build.gradle:56]: Versions can go into the project gradle.

File: app/src/main/java/com/jyoti/androidtutorial/home/data/data_source/HomeRemoteDataSource.kt [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+import com.jyoti.androidtutorial.home.data.model.HomeResponseModel
+import retrofit2.Call
+import retrofit2.http.GET
+
+
+interface HomeRemoteDataSource {
+
+    @GET("/home")
+    fun fetchHomeDataRemote(): Call<HomeResponseModel?>?
```
Comment (PrateekDev121) [inline app/src/main/java/com/jyoti/androidtutorial/home/data/data_source/HomeRemoteDataSource.kt]: can the Call<> be null ???

**Review by PrateekDev121 — COMMENTED** [review]
Good project for beginners. This project will help in clearing core concepts

File: app/build.gradle (hunk 1 above)
Comment (JyotiBhambhu) [reply app/build.gradle:56]: Yes, it is best practice to keep the version in a separate file, and that's why I put compose version in project.gradle. But as this is only a basic setup and for beginners, I am not paying attention to scalability and modularity for time's sake. But will come back later on these topics one by one.

Comment (PrateekDev121) [reply app/build.gradle:56]: great

**Review by PrateekDev121 — APPROVED** [review]
good start

File: app/src/main/java/com/jyoti/androidtutorial/home/data/data_source/HomeRemoteDataSource.kt (hunk 2 above)
Comment (JyotiBhambhu) [reply app/src/main/java/com/jyoti/androidtutorial/home/data/data_source/HomeRemoteDataSource.kt]: No, it can not be null. Thank you for pointing this.

**Pull request merged**